docker-compose/docker-compose.base.yml

```diff
@@ -1,56 +1,51 @@
-version: '3'
-
+version: '3.9'
 services:
-
   case-server:
     image: powsybl/case-server:latest
     ports:
       - 5000:80
     volumes:
-      - $PWD/../../k8s/resources/common/config/case-server-application.yml:/config/specific/application.yml:Z
-      - $PWD/../../k8s/resources/common/config/common-application.yml:/config/common/application.yml:Z
-      - $GRIDSUITE_DATABASES/cases:/cases:Z
+      - ./../k8s/resources/common/config/case-server-application.yml:/config/specific/application.yml:ro
+      - ./../k8s/resources/common/config/common-application.yml:/config/common/application.yml:ro
+      - ${GRIDSUITE_DATABASES:-/tmp/gridsuite}/cases:/cases
```

Comment:
Why did you have to change this line and add `-/tmp/gridsuite` ?

Reply:
will be in another PR as asked